src/scouter/ScouterConstants.ts

```diff
@@ -0,0 +1,2 @@
+export const serverHostName = "192.168.1.126";
```

Comment:
This is used for client communication to the server, you will need to get the servers ip from runtime.